src/ballet/reedsol/fd_reedsol_pi.c

```diff
@@ -68,22 +68,7 @@
 
 #include "../../util/simd/fd_sse.h"
 
-#define ws_t __m256i
-#define ws_add(a,b)         _mm256_add_epi16( (a), (b) )
-#define ws_sub(a,b)         _mm256_sub_epi16( (a), (b) )
-#define ws_bcast(s0)        _mm256_set1_epi16( (s0) )
 #define ws_adjust_sign(a,b) _mm256_sign_epi16( (a), (b) ) /* scales elements in a by the sign of the corresponding element of b */
-#define ws_mullo(a,b)       _mm256_mullo_epi16( (a), (b) )
-#define ws_mulhi(a,b)       _mm256_mulhi_epu16( (a), (b) )
```

Comment:
@ptaffet-jump fd_avx_ws.h uses `_mm256_mulhi_epi16` (sign extend a, b before multiplying) 
What should we do here? Maybe just change reedsol_pi.c to use wh_mulhi?

Reply:
I think reedsol should be using ushort, but I'm not sure why it says short everywhere. I need to try to refresh my memory.

Reply:
@ptaffet-jump You probably had ws=>ushort when you wrote the code. I picked ws=>short and wh=>ushort when I wrote my wrappers. Then I realized you already picked ws. Please let me know what we should do about this

Reply:
Changed fd_reedsol_pi to use wh_mulhi